sql_execer.go

```diff
@@ -18,10 +18,13 @@ type wExecer struct {
 func (conn *wExecer) Exec(query string, args []driver.Value) (driver.Result, error) {
```

Comment:
Query: I see that startSQLSpan() was moved to post the actual query interfaces. Wouldn't this cause inaccurate database query execution durations?

Reply:
> Query: I see that startSQLSpan() was moved to post the actual query interfaces. Wouldn't this cause inaccurate database query execution durations?

I am not sure if I understood the question.
Moved from where to where?
What do you mean by actual query interfaces? 
The idea was to make the function compliant to detect specific databases. As far as I can see there was no change on how startSQLSpan() effectively works, but I may have overlooked something.
Could you please elaborate?

Reply:
As discussed, the span should start before the operation.

Reply:
Yes, thanks for the clarification. That was a very good catch.
Changes are ready for another round of review :)